frank-flow/src/frontend/src/app/shared/components/edit-dialog/edit-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<ngx-smart-modal
+  #editDialog
+  (onDataAdded)="onDataAdded()"
+  (onOpenFinished)="inputElement.focus()"
+  (onAnyCloseEvent)="edit()"
+  class="editDialog"
+  customClass="nsm-centered"
+  identifier="editDialog"
+>
+  <div>
+    <h2>
+      Modify {{ currentDirectory?.path ? 'folder' : 'file' }} {{ oldFileName }}
```

Comment:
Add a colon so the name and type is clear:
`Modify {{ currentDirectory?.path ? 'folder' : 'file' }}: {{ oldFileName }}`